numbat-cli/src/main.rs

```diff
@@ -169,16 +204,25 @@ impl Cli {
             }
         }
 
-        if load_prelude {
+        if load_prelude
+            && self.config.exchange_rate_fetching_policy != ExchangeRateFetchingPolicy::DontFetch
+        {
             {
                 self.context
                     .lock()
                     .unwrap()
                     .load_currency_module_on_demand(true);
             }
-            thread::spawn(move || {
-                numbat::Context::prefetch_exchange_rates();
-            });
+
+            match self.config.exchange_rate_fetching_policy {
+                ExchangeRateFetchingPolicy::Prefetch => {
+                    thread::spawn(move || {
+                        numbat::Context::prefetch_exchange_rates();
+                    });
+                }
+                ExchangeRateFetchingPolicy::FetchOnFirstUse
+                | ExchangeRateFetchingPolicy::DontFetch => {}
+            }
```

Comment:
Let's replace that with a simple `if self.config.exchange_rate_fetching_policy ==                  ExchangeRateFetchingPolicy::Prefetch`?

Reply:
That's what I did initially, but then I changed it to this so that we'd be forced to account for any new/renamed values of `ExchangeRateFetchPolicy`, but it's probably unnecessary, I agree.